### ISSUE TITLE
Register a User

### ISSUE THREAD

**Issue by hannahhall** [cseward1/NSSBackEndProjectCas#3]
**Once the user lands on the Register Page...**

The user will see a Registration form:
- create a form
- this form will require: First Name, Last Name, Picture,  Email, & password
**Stretch Goal After I hit MVP:** Ask them what genre, authors they like and play with that information when they are redirected to the home page.

* Make sure all forms that are required are validated
**Required Information:**
- First Name
- Last Name
- Email
- Password

-Store that information with your SQL connection when the user clicks an affordance to **Submit** the form. 

If the form was filled out correctly and submitted, **the user will be redirected to the home page and allowed to start personalizing their account and find friends.**
